packages/account-v2/.eslintrc.json

```diff
@@ -0,0 +1,74 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
```

Comment:
```suggestion
    jest: true,
    },
```

Reply:
Can u explain what it does? I didn't quite understand from [this](https://eslint.org/docs/latest/use/configure/language-options#specifying-environments)

Reply:
If I'm not mistaken, eslint will argue on jest methods. we have this option in our deriv.app config as well

Reply:
Done